src/main/kotlin/blackjack/model/result/PlayerResultTracker.kt

```diff
@@ -1,13 +1,13 @@
 package blackjack.model.result
 
 class PlayerResultTracker : ResultTracker {
-    private var lastResult: Result? = null
+    var result: Result = Result.UNKNOWN
 
     override fun toString(): String {
-        return lastResult?.description ?: "No result recorded."
+        return result?.description ?: "No result recorded."
```

Comment:
The null operator is no longer needed.
Also, if `toString()` is being used for the View, consider moving that responsibility to the View layer instead. (Same feedback as before) 🙂